src/sync_wrapped.rs

```diff
@@ -0,0 +1,4 @@
+unsafe impl Sync for EventInstance {}
+unsafe impl Send for EventInstance {}
+
+pub struct EventInstance(pub libfmod::EventInstance);
```

Comment:
nit: When this file grows, I think it makes more sense to

1. swap the train impls with the struct definition
2. move EventInstance into `src/sync_wrapped/event_instance.rs